### ISSUE TITLE
Settings design review

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/desktop#5142]
Notes from the meeting @claucambra @mgallien and cc @tobiasKaminsky for info

* [x] Use "Capital Case" only for headings and buttons, "Sentence case" for rest
* [ ] Use operating system native icons for "General" and "Network" (native icons for each OS)

## General
* [x] Remove "Using virtual files plugin: suffix" line → move to debug archive
* [x] Remove "osx-22.1.0" / "ubuntu-5.15 …" kernel header info → move to debug archive
* [x] Simplify About / Updates into 1 section
* [x] Proposed combination of About & Updates section – `( These are buttons )`
> **Info**
>
> Nextcloud desktop client 3.6.1
>
> Update channel ( stable ⏷ )
>
> * [x] Automatically check for updates ( Check now )
>
> ( Usage documentation ) ( Legal notice )

## Network
* [x] Only show detailed proxy input fields if you set the "Specifcy proxy manually as" – same for Bandwidth limiter
* [x] Width of fields can be reduced, HTTPS proxy dropdown, hostname, user and password – don’t need to extend full width
* [x] Elements should be directly following the text: HTTPS proxy dropdown, inputs after "Limit to" text

## Probably blocked by move to QML

* [ ] Section headings: title is smaller than normal text, not really connected to box
* [ ] Network settings: Vertical spacing of radio buttons not consistent (probably because of text field on 3rd line)

**Comment by tobiasKaminsky** [comment]
@claucambra can you give me an estimation how long this will take?
If one ticket is too big for 2 weeks, please split it up.

**Comment by allexzander** [comment]
this was implemented in https://github.com/nextcloud/desktop/pull/6150
